### PR TITLE
Namespace declaration is never used

### PR DESCRIPTION
**Pull request by neeldoshii**
Removed the namespace which are declared but not used from string and xml files.

## Purpose / Description
Fixed the Unused Namespace Declaration Android Studio warnings.

## Fixes
Related to #13282 

## Approach
_How does this change address the problem?_

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by BrayanDSO** [comment]
I believe that the locales XML files are automatically generated, so there is no value in changing them directly.

Also, this issue can go on forever since someone will always forget to remove it. Unless we start using the headless IDE linter proposed at #13282, I don't think this is worth reviewing

**Review by ShridharGoel — APPROVED** [review]

**Comment by neeldoshii** [comment]
Thank you @ShridharGoel for approving the PR.

File: AnkiDroid/src/main/res/layout/locale_dialog_fragment_textview.xml [hunk 1]
```diff
@@ -14,12 +14,9 @@
...
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<androidx.appcompat.widget.AppCompatTextView xmlns:android="http://schemas.android.com/apk/res/android"
     android:id="@+id/locale_dialog_fragment_text_view"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:textSize="@dimen/content_textsize"
-    android:padding="@dimen/content_vertical_padding"
-    />
\ No newline at end of file
+    android:padding="@dimen/content_vertical_padding" />
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/layout/locale_dialog_fragment_textview.xml:22]: it is difficult to review the 101 files changed when more than removing the namespace is happening

here spacing is changed which makes me think "oh now was `android:padding` also removed?"

only on close examination (read as: after demanding time from a busy reviewer) is it possible to see that this line did not have a semantic difference

File: AnkiDroid/src/main/res/layout/locale_dialog_fragment_textview.xml [hunk 2]
```diff
@@ -14,12 +14,9 @@
   You should have received a copy of the GNU General Public License along with
   this program.  If not, see <http://www.gnu.org/licenses/>.
   -->
-<androidx.appcompat.widget.AppCompatTextView
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<androidx.appcompat.widget.AppCompatTextView xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/layout/locale_dialog_fragment_textview.xml:17]: spacing change, muddies the diff, harder to review

File: AnkiDroid/src/main/res/layout/navigation_drawer_layout.xml [hunk 3]
```diff
@@ -1,9 +1,8 @@
...
 <androidx.drawerlayout.widget.ClosableDrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                                           xmlns:app="http://schemas.android.com/apk/res-auto"
-                                           android:id="@+id/drawer_layout"
-                                           android:layout_width="match_parent"
-                                           android:layout_height="match_parent"
-                                           android:fitsSystemWindows="true">
+    android:id="@+id/drawer_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:fitsSystemWindows="true">
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/layout/navigation_drawer_layout.xml:6]: etc - -5 +4 diff, but it should just be -1

File: AnkiDroid/src/main/res/layout-land/introduction_layout.xml [hunk 4]
```diff
@@ -15,10 +15,8 @@
   ~  this program.  If not, see <http://www.gnu.org/licenses/>.
   -->
 
-<androidx.constraintlayout.widget.ConstraintLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/layout-land/introduction_layout.xml:18]: this spacing change should be reverted - there is no reason to have a stack of things formatted nicely but with one-and-only-one snuggled up on the tag opening line

File: AnkiDroid/src/main/res/layout/card_browser.xml [hunk 5]
```diff
@@ -1,11 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<androidx.coordinatorlayout.widget.CoordinatorLayout
-    android:id="@+id/root_layout"
+<androidx.coordinatorlayout.widget.CoordinatorLayout android:id="@+id/root_layout"
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/layout/card_browser.xml:3]: same, please revert this

File: AnkiDroid/src/main/res/layout/locale_dialog_fragment_textview.xml (hunk 1 above)
Comment (mikehardy) [inline AnkiDroid/src/main/res/layout/locale_dialog_fragment_textview.xml:22]: unnecessary spacing change

File: AnkiDroid/src/main/res/layout/reviewer_fullscreen_noanswers.xml [hunk 6]
```diff
@@ -1,5 +1,4 @@
-<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/root_layout"
+<androidx.coordinatorlayout.widget.CoordinatorLayout android:id="@+id/root_layout"
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/layout/reviewer_fullscreen_noanswers.xml:1]: prefer to just get rid of the xmlns:tools line snuggled up on tag open vs moving the android:id up there, one is a clean 1-line diff, other is -2 +1

**Review by mikehardy — CHANGES_REQUESTED** [review]
this seems okay, and I believe a compilation test is enough to prove it works, but by including irrelevant spacing changes in the diff, it's more difficult to review.

I left a few specific comments on things to revert and I recognize it is being really picky, but the way you do one thing is the way you do all things, and we want to be in the habit in this repo of never proposing diffs with unnecessary spacing changes unless they are in a clean+separate commit. We value reviewer time more than that

**Comment by david-allison** [comment]
Optional: My understanding is that if we make the change in the "main" file, then the translation sync process will remove it when it next occurs

This would massively cut down on the changes in this PR

**Pull request closed without merging**